operator_pack.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+print_help_message(){
+    echo "Usage:
+ operator_pack.sh -v [Openshift Version] -o [Operator Version] -r [Registry FQDN] -u [RedHat Account User] -p [Password] -n [Package Name] -i [Operator Index Name] -b [Base Directory] 
```

Comment:
See if you can pull the latest operator version as default, without manually checking the latest version.